include/z64save.h

```diff
@@ -278,13 +278,7 @@ typedef struct SaveInfo {
     /* 0xEA8 */ u32 unk_EA8[2];                        // Related to blue warps
     /* 0xEB0 */ u32 stolenItems;                       // Items stolen by Takkuri and given to Curiosity Shop Man
     /* 0xEB4 */ u32 unk_EB4;
-    /* 0xEB8 */ u32 bankRupees;
-    /* 0xEBC */ u32 unk_EBC;
-    /* 0xEC0 */ u32 unk_EC0;                           // Fishing flags
-    /* 0xEC4 */ u32 unk_EC4;
-    /* 0xEC8 */ u32 horseBackBalloonHighScore;
-    /* 0xECC */ u32 lotteryCodeGuess;                  // Lottery code chosen by player (only uses lower three hex digits)
-    /* 0xED0 */ u32 shootingGalleryHighScores;         // High scores for both shooting galleries. Town uses lower 16 bits, Swamp uses higher 16 bits.
```

Comment:
I think these comments could be moved to the enum you made? Just to preserve this info